package/CHANGELOG

```diff
@@ -75,6 +75,7 @@ Fixes
     *exluding* its upper boundaries) (Issue #2200, PR #2201)
   * *Group.wrap() is now guaranteed to only work on atoms actually belonging to
     the calling group. (Issue #2203, PR #2204)
+  * Fixed documentation in analysis/contacts.py
```

Comment:
Since the changelog is intended for users, it's better to specify the module instead of the file:
```suggestion
  * Fixed documentation of MDAnalysis.analysis.contacts
```